src/Avalonia.Controls.ColorPicker/ColorSpectrum/ColorSpectrum.cs

```diff
@@ -1201,20 +1185,29 @@ private async void CreateBitmapsAndColorMap()
                 {
                     case ColorSpectrumComponents.HueValue:
                     case ColorSpectrumComponents.ValueHue:
+                        _saturationMinimumBitmap?.Dispose();
                         _saturationMinimumBitmap = _minBitmap;
+                        _saturationMaximumBitmap?.Dispose();
                         _saturationMaximumBitmap = _maxBitmap;
                         break;
                     case ColorSpectrumComponents.HueSaturation:
                     case ColorSpectrumComponents.SaturationHue:
+                        _valueBitmap?.Dispose();
                         _valueBitmap = _maxBitmap;
                         break;
                     case ColorSpectrumComponents.ValueSaturation:
                     case ColorSpectrumComponents.SaturationValue:
+                        _hueRedBitmap?.Dispose();
```

Comment:
Hm... I don't have anything against using a type other than ArrayList. However, Dispose() implies more native resource usage and shouldn't really be needed in a managed environment. I don't like the dispose calls everywhere and these things are easy to forget about in other code paths...

Reply:
It should, really. We can leave bitmap to float in the memory, as they will be garbage collected anyway. But from our experience we know that skia objects finalizers are pretty slow, especially when there are many of them. Since we can dispose old value with not much issues - it should be done.